README.md

```diff
@@ -23,3 +23,38 @@ This Turborepo has some additional tools already setup for you:
 - [TypeScript](https://www.typescriptlang.org/) for static type checking
 - [ESLint](https://eslint.org/) for code linting
 - [Prettier](https://prettier.io) for code formatting
+
+## Usage
+
+Install, and run both apps simultaneously
+
+```bash
+npm install && npm run dev
+```
+
+Or run apps separately
+
+```bash
+npm run dev:features
+```
+
+```bash
+npm run dev:mvp
+```
+
+### Docker
+
```

Comment:
Joku tälläinen tähän johonkin väliin:

### Docker requirements

The following network must be created: `vfd-network`.

Create the network with command:

```
docker network create vfd-network
```

Reply:
Tämä johtuu siitä että projektin docker-compose-filet vaatii tuota vfd-nettiä (eli se pitää olla olemassa vaikka ei käyttäisikään esim. sitä traefikkia)